src/LettuceEncrypt/IFailedOrderStore.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Archon Systems.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace LettuceEncrypt;
+
+/// <summary>
+///
+/// </summary>
+internal interface IFailedOrderStore
```

Comment:
Decided not to add this to the public api for now, in memory is probably good enough for most use cases - it definitely is for us